gen-windows.bat

```diff
@@ -1,5 +0,0 @@
-@echo off
-
-REM Generate project files for your last Visual Studio version you have
-call cmake -S . -B build-windows -DCMAKE_TOOLCHAIN_FILE=./cmake/toolchain/windows-x64.cmake
```

Comment:
no delete my byatch please... is just easy to setup, keep it here

Reply:
What is it hard about literally just opening Visual Studio, selecting the Vit3K folder right in the welcome window, selecting the configure preset (if not already selected) and using Project > Configure Cache? You can do anything right in Visual Studio without needing a batch file:

https://user-images.githubusercontent.com/48522412/169663107-1ab72c02-fde1-401e-a5d0-083d5be388af.mp4

Reply:
too long sorry lol, i thinks you no understund, i have 20 folder of vita3k for branch xD
i have never using this, i go inside my rep, i launch batch, and i launch sln file, that all

select folder any time ugh nop lol

Reply:
I'll have to leave them then.